firefox-settings/pure.js

```diff
@@ -155,3 +155,37 @@ pref("browser.safebrowsing.provider.mozilla.pver", 0, locked);
 pref("browser.safebrowsing.provider.mozilla.reportURL", "", locked);
 pref("browser.safebrowsing.provider.mozilla.updateURL", "", locked);
 pref("browser.safebrowsing.reportPhishURL", "", locked);
+
+pref("browser.contentblocking.category", "strict"); // https://blog.mozilla.org/security/2021/02/23/total-cookie-protection/
+pref("browser.download.autohideButton", false, locked); //do not hide download button automatically
+pref("browser.download.start_downloads_in_tmp_dir", true, locked); //start downloads in temp
+pref("browser.helperApps.deleteTempFileOnExit", true); // https://bugzilla.mozilla.org/show_bug.cgi?id=238789#c19
+pref("browser.tabs.unloadOnLowMemory", true, locked); // Unload unused tabs
+pref("browser.uitour.enabled", false); // https://trac.torproject.org/projects/tor/ticket/19047#comment:3
+
+pref("content.notify.interval", 100000); // page reflow timer, lower redrawn rendering timer, increases responsiveness but increase total load time
+pref("dom.security.https_only_mode", true, locked); // only allow https in all windows, including private browsing
+pref("network.auth.subresource-http-auth-allow", 1, locked); // block HTTP authentication credential dialogs
+pref("network.cookie.sameSite.noneRequiresSecure", true); // https://hacks.mozilla.org/2020/08/changes-to-samesite-cookie-behavior/
+pref("network.dns.disablePrefetch", true); // https://developer.mozilla.org/docs/Web/HTTP/Headers/X-DNS-Prefetch-Control
```

Comment:
This is already disabled by policies.

Reply:
Alternatively you could also remove `NetworkPrediction` from the policies and leave it here, as I'm not sure if it should be locked.

Reply:
Removed as it's in policies set by you